### PR TITLE
Features/list

### PR DESCRIPTION
**Pull request by SHIKHASINGH1506**
Added 3 components in this branch(as rating and typography had comparatively less code)
Modified document by changing the code snippet
Modified typographic a bit

### CODE REVIEW

File: utility.css [hunk 1]
```diff
@@ -50,8 +72,28 @@
   margin-bottom: 1rem;
 }
 .my-6 {
-  margin-top: 2rem;
-  margin-bottom: 2rem;
+  margin-top: 1.5rem;
+  margin-bottom: 1.5rem;
+}
+.mr-2{
```
Comment (tushargoyal812) [inline utility.css:78]: Suggestion: add space before { it is good for indentation.

File: utility.css [hunk 2]
```diff
@@ -114,3 +154,12 @@ p {
   margin: 0.625rem 0;
   line-height: 1.5;
 }
+.text-start{
```
Comment (tushargoyal812) [inline utility.css:157]: same here

**Review by tushargoyal812 — COMMENTED** [review]
everything looks good only take care of indentation on some places.

File: Components/list.css [hunk 3]
```diff
@@ -0,0 +1,30 @@
+.list-group {
+  display: flex;
+  flex-direction: column;
```
Comment (devdeadviz) [inline Components/list.css:3]: Just a suggestion, you can create a different stylesheet for layouts.

File: Components/list.css [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+  background-color: white;
+  max-width: 12.5rem;
+  width: 12.5rem;
+}
+.list-item {
+  display: flex;
+  align-items: center;
```
Comment (devdeadviz) [inline Components/list.css:10]: same here

**Review by devdeadviz — COMMENTED** [review]
Added some suggestions, have a look at it.

**Pull request merged**